README.md

```diff
@@ -13,3 +13,45 @@ For installation instructions see [INSTALL.md](https://github.com/hypothesis/pip
 
 For how to set up a pip-sync-faster development environment see
 [HACKING.md](https://github.com/hypothesis/pip-sync-faster/blob/main/HACKING.md).
+
+pip-sync-faster makes
+[pip-sync](https://pip-tools.readthedocs.io/en/latest/#example-usage-for-pip-sync)
+run faster in the case where there's nothing to do because the virtualenv is
+already up to date with the requirements files. On my machine, with my
+requirements files, it shaves off over 500ms in the time taken to run pip-sync:
+
+```terminal
+$ time pip-sync requirements/foo.txt
+Everything up-to-date
+
+real    0m0.569s
+user    0m0.525s
+sys     0m0.045s
+
+$ time pip-sync-faster requirements/foo.txt
+
+real    0m0.037s
+user    0m0.029s
+sys     0m0.008s
+```
+
+pip-sync-faster does this by saving hashes of the given requirements files in a
```

Comment:
```suggestion
`pip-sync-faster` does this by saving hashes of the given requirements files in a
```
? Might stop the names from blending in quite so much?

Reply:
:+1: